Content.Server/_Impstation/Tourist/TouristConditionsSystem.cs

```diff
@@ -0,0 +1,83 @@
+using Content.Server.Objectives.Components;
+using Content.Server.Roles;
+using Content.Server._Impstation.Tourist.Components;
+using Content.Shared.Objectives.Components;
+using Content.Shared._Impstation.Tourist.Components;
+using Content.Shared.Roles;
+using Content.Shared.Warps;
+using Content.Shared.Mind;
+using Robust.Shared.Random;
+using System.Linq;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Paper;
+using Robust.Shared.Containers;
+
```

Comment:
```suggestion
using Content.Server.Objectives.Systems;

```
needs this refrence for numbering objectiv to work